File: info.go
package main

import (
	"fmt"
	"os"
	"runtime"
)

func main() {
	if runtime.GOOS == "windows" {
		fmt.Print("\033]0;⚡ KilixKilik Profile\007")
	}

	fmt.Print("\033[36m")
	fmt.Print(`
    __  ___      __       __  __      
   /  |/  /___ _/ /______/ /_/ /____ _
  / /|_/ / __ \`/ __/ ___/ __/ __/ __ \`/
 / /  / / /_/ / /_(__  ) /_/ /_/ /_/ / 
/_/  /_\`,_,_/\__/____/\__/\__/\`,_,_/  
`)
	fmt.Print("\033[0m")

	fmt.Print("\033[32m")
	fmt.Print(`
 ✦ Имя: Кирилл (KilixKilik)
 ✦ Язык: Python 🐍
 ✦ GitHub: https://github.com/KilixKilik
 ✦ ОС: Windows 10 💻
`)
	fmt.Print("\033[0m")

	fmt.Print("\033[33m 🚀 Нажми Enter, чтобы выйти...\033[0m\n")
	var input string
	fmt.Scanln(&input)
}

File: info.c
#include <windows.h>
#include <stdio.h>

int main() {
    SetConsoleTitleA("⚡ KilixKilik Profile");
    HANDLE hConsole = GetStdHandle(STD_OUTPUT_HANDLE);

    SetConsoleTextAttribute(hConsole, FOREGROUND_BLUE | FOREGROUND_INTENSITY);
    printf(
        "    __  ___      __       __  __      \n"
        "   /  |/  /___ _/ /______/ /_/ /____ _\n"
        "  / /|_/ / __ `/ __/ ___/ __/ __/ __ `/\n"
        " / /  / / /_/ / /_(__  ) /_/ /_/ /_/ / \n"
        "/_/  /_/\\__,_/\\__/____/\\__/\\__/\\__,_/  \n\n"
    );

    SetConsoleTextAttribute(hConsole, FOREGROUND_GREEN | FOREGROUND_INTENSITY);
    printf(
        " ✦ Имя: Кирилл (KilixKilik)\n"
        " ✦ Язык: Python \xF0\x9F\x90\x8D\n"
        " ✦ GitHub: https://github.com/KilixKilik\n"
        " ✦ ОС: Windows 10 \xF0\x9F\x92\xBB\n\n"
    );

    SetConsoleTextAttribute(hConsole, FOREGROUND_YELLOW | FOREGROUND_INTENSITY);
    printf(" 🚀 Нажми Enter, чтобы выйти...\n");
    getchar();
    return 0;
}
